src/index/diskann/diskann.cc

```diff
@@ -380,10 +376,12 @@ DiskANNIndexNode<T>::Deserialize(const BinarySet& binset, const Config& cfg) {
 
     pq_flash_index_ = std::make_unique<diskann::PQFlashIndex<T>>(reader, diskann_metric);
 
-    auto load_expect =
-        TryDiskANNCall<int>([&]() -> int { return pq_flash_index_->load(pool_->size(), index_prefix_.c_str()); });
-
-    if (!load_expect.has_value() || load_expect.value() != 0) {
+    if (TryDiskANNCall([&]() {
```

Comment:
suggest not change this code

Reply:
Chatted offline, extracted the lambda into be a variable for better readability